Example_MeliCardDrawer/Podfile

```diff
@@ -1,6 +1,6 @@
 source 'https://github.com/CocoaPods/Specs.git'
 
-platform :ios, '9.0'
+platform :ios, '10.0'
```

Comment:
I changed minimun platform cause of some dependencies not been compatible anymore with ios 9